k8s/charts/openebs/README.md

```diff
@@ -91,6 +91,7 @@ The following table lists the configurable parameters of OpenEBS chart and their
 | `ndm.sparse.count`                      | Number of sparse files to be created        | `1`                                               |
 | `ndm.sparse.filters.excludeVendors`     | Exclude devices with specified vendor       | `CLOUDBYT,OpenEBS`                                |
 | `ndm.sparse.filters.excludePaths`       | Exclude devices with specified path patterns| `loop,fd0,sr0,/dev/ram,/dev/dm-`                  |
+| `ndm.updateStrategy.type`               | Update strategy policy                      | `RollingUpdate`                                   |
 | `jiva.image`                            | Docker Image for Jiva                       | `quay.io/openebs/jiva`                            |
 | `jiva.imageTag`                         | Docker Image Tag for Jiva                   | `0.8.0`                                           |
```

Comment:
Should all the descriptions/defaults be changed to 0.8.1?

Reply:
Ahh..  forget to change them in README .. fixed !!